apiary.apib

```diff
@@ -518,6 +518,28 @@ Get a list of all the definitions of custom fields.
 
     + Attributes (object)
         + page (Page, optional)
+        + filter (object, optional)
+            + context (enum[string], optional)
+                + Members
+                    + contact
+                    + company
+                    + form
+                    + event
+                    + product
+                    + project
+                    + milestone
+                    + sale
```

Comment:
B: This one should be `deal`

Reply:
@mrtus - This is actually correct like is imo. Please check the for column in the crm_custom_fields table to see why

Reply:
You can also take a look at the CustomFieldDefinitionContext value object in core